app/models/warning.rb

```diff
@@ -3,6 +3,16 @@ class Warning < Tag
   NAME = ArchiveConfig.WARNING_CATEGORY_NAME
   index_name Tag.index_name
 
+  DISPLAY_NAME_MAPPING = {
+    ArchiveConfig.WARNING_DEFAULT_TAG_NAME => ArchiveConfig.WARNING_DEFAULT_TAG_DISPLAY_NAME,
+    ArchiveConfig.WARNING_NONE_TAG_NAME => ArchiveConfig.WARNING_NONE_TAG_DISPLAY_NAME,
+    ArchiveConfig.WARNING_SOME_TAG_NAME => ArchiveConfig.WARNING_SOME_TAG_DISPLAY_NAME,
```

Comment:
Do we still need this? We got rid of the "Some Archive Warnings Apply" option....idk, before I came onboard, so a long time ago. I think this is the only remaining reference to it; I don't even see anything in the config file: https://github.com/otwcode/otwarchive/blob/master/config/config.yml#L158

 If we have anything in the database still using this warning, we should probably do an announcement and clean it up.

Reply:
I can get rid of it if you want, sure.

Reply:
Let's do that and call this ready to merge 👍